README.md

```diff
@@ -1,3 +1,10 @@
 # Simple irc bot written in golang
 ## This bot is currently going to function on #osdg-iiith on freenode.
 This is a small irc bot written in golang, you're welcome to add functionality to it!
+
+Instructions:
```

Comment:
I feel like these instructions could be better phrased. Example: "set path to /bin and then execute the 'bot' executable". There's a better way to say that.